src/refract/Element.h

```diff
@@ -566,6 +566,8 @@ namespace refract
             hasContent = true;
             value.push_back(e);
         }
+
+        void addMember(const std::string& key, IElement* val, IElement::renderFlags render = IElement::rDefault);
```

Comment:
I suggest to avoid add this member function, instead you can do:
```
object->push_back(new MemberElement(key, val, renderFlag));
```
Meaning of this is obvious and there is no so much additional writing.

Next argument is, we are not compatible among other kind of